app/Controller/PagesController.php

```diff
@@ -84,79 +102,82 @@ public function display17() {
  * @throws NotFoundException When the view file could not be found
  *	or MissingViewException in debug mode.
  */
-	public function display() {
-    $eeeeeeeeee;
-    $aaaaaaaaaa;
-    $bbbbbbbbbb;
-    $aaaaaaaaaa;
+    public function display()
```

Comment:
__[staging][phpmd]__
* The variable $title_for_layout is not named in camelCase.
* The variable $title_for_layout is not named in camelCase.